packages/launch/src/focused-launch/success/style.scss

```diff
@@ -0,0 +1,112 @@
+@import '~@automattic/onboarding/styles/variables';
+@import '~@automattic/onboarding/styles/mixins';
+
+.focused-launch-success {
+	padding: 0 20px 120px;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+}
+
+.focused-launch-success__wrapper {
+	width: 100%;
+	max-width: 420px;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	text-align: center;
+
+	.onboarding-title {
+		margin-bottom: 12px;
+	}
+
+	.onboarding-subtitle {
+		margin-bottom: 32px;
+	}
+}
+
+.focused-launch-success__wordpress-icon {
+	position: absolute;
+	top: 0;
+	left: 0;
+}
+
+.focused-launch-success__confetti {
+	width: 100%;
+	max-width: 356px;
+	margin-bottom: -20px;
+}
+
+.focused-launch-success__url-wrapper,
+.focused-launch-success__url-copy-button {
+	// @TODO: why can't border-radius be 4px?
+	border-radius: 4px;
+	border: 1px solid var( --studio-gray-5 );
+}
+
+.focused-launch-success__url-wrapper {
+	display: flex;
+	align-items: center;
+	padding: 6px 6px 6px 16px;
+	width: 100%;
+	margin-bottom: 24px;
+
+	@include break-small {
+		width: auto;
+		max-width: 100%;
+		margin-bottom: 32px;
+	}
+}
+
+.focused-launch-success__url-field {
+	@include onboarding-small-text();
+	color: var( --studio-gray-60 );
+	flex-grow: 0;
+	flex-shrink: 1;
+	min-width: 0;
+	overflow: hidden;
+	white-space: nowrap;
+	text-overflow: ellipsis;
+}
+
+.focused-launch-success__url-link {
+	fill: var( --studio-gray-50 );
+	flex-grow: 0;
+	flex-shrink: 0;
+	margin-left: 4px;
+	margin-right: 16px;
+	font-size: 0;
+	line-height: 1;
+
+	@include break-small {
+		margin-right: 24px;
+	}
+}
+
+.focused-launch-success__url-copy-button {
+	@include onboarding-small-text();
+	color: var( --studio-gray-60 );
+	// @TODO: why can't font-weight be 500%
```

Comment:
I think it's because we wanted to have the styles aligned with Gutenberg (see https://github.com/Automattic/wp-calypso/issues/47161 for example).

Here's the conclusion from a similar discussion in the context of Focused Launch about headings: https://github.com/Automattic/wp-calypso/pull/46784#discussion_r513173063
So I think it's fine to keep using 400 for now until we do a makeover across the onboarding flows.